src/pydocstyle/config.py

```diff
@@ -208,7 +208,10 @@ def _get_config(self, node):
         if path in self._cache:
             return self._cache[path]
 
-        config_file = self._get_config_file_in_folder(path)
+        if self._options.config_file:
```

Comment:
I take back my wanna-test-my-code-on-another-project's-configuration, it does seem a little pointless 😋 
+1 @Nurdok